Gemfile

```diff
@@ -25,7 +25,8 @@ group :development, :unit_tests do
 end
 
 group :system_tests do
-  gem 'beaker',               :require => false
+  gem 'beaker', :git => 'https://github.com/trevor-vaughan/beaker.git', :branch => 'BKR-978-2.51.0'
```

Comment:
We're managing this with modulesync, so we might need to use this in all of our modules...